display.js

```diff
@@ -90,7 +91,7 @@ chrome.runtime.onMessage.addListener(function(request, sender, sendResponse) {
             () => sendResponse({
                 message: "FAILED"
             })
-        )
+        );
```

Comment:
we should agree on semicolons or not then find a way to enforce it. what do you think?

Reply:
Yup, weirdest thing about javascript and python. I am fine with either, what do you think? I tried to find if you were using either but couldn't find a specific style

Reply:
lol yeah, cause I didn't decide on one. haha. I could go either way. I did semi's in the past but think I'm leaning towards none.

Reply:
None it is

Reply:
Ill look in sc more later though and remove them / add some enforcing contraints on another PR

Reply:
Awesome